### DISCUSSION TITLE
Nvidia shutdown zluda development whats next for this project?

### DISCUSSION THREAD

**Discussion by VeteranXT** [lshqqytiger/stable-diffusion-webui-directml#518] [General]
Will this fork be closed by DMCA? or all  related functions to zluda?

**Comment by CS1o** [comment]
Nvidia didnt shutdown ZLUDA. It was rolled back on AMDs request.
You cand find that Information here:
https://github.com/vosen/ZLUDA?tab=readme-ov-file#what-happened

This repo will continue as it has nothing to do with zluda other than supporting it and also lshqqtiger is still maintaining and working on ZLUDA.

**Reply by VeteranXT** [reply]
Old question but I do follow that fork xD
